docs/rfcs/0000-new-pipeline.md

```diff
@@ -0,0 +1,252 @@
+- Start Date: 2020-11-18
+- Target Major Version: 2.0
+- Reference Issues: (leave this empty)
+- Implementation PR: (leave this empty)
+- Author: @yorkie
+
+# Summary
+
+Introducing a new Pipeline syntax for supporting:
+
+- multiple nodes for the same plugin.
+- model deployment and distribution.
+
+# Basic example
+
+```json
+{
+  "specVersion": "2.0",
+  "plugins": [
```

Comment:
what if the user specifies more than one dataCollect/dataAccess/modelDeinfe plugins? Shall we make the whole plugins to be an array or just dataProcess to be an array?

Reply:
To make sure the extensibility of plugins, I think we could keep all plugins should be in the array, how about add additional rules to restrict the array such as limit only 1 "modelDefine" type?

Reply:
This sounds good. We can modify the schema so that the numbers are limited to 1 for now.

Reply:
@rickycao-qy I have updated the schema below, the plugin items are split into multiple sub-schemas, which I defined the `maxItems` and `minItems` in different ones.